cpp/dolfin/common/IndexMap.h

```diff
@@ -100,6 +102,10 @@ class IndexMap
   /// Owner rank (on global communicator) of each ghost entry
   Eigen::Array<std::int32_t, Eigen::Dynamic, 1> ghost_owners() const;
 
+  /// Create a map from shared indices to sharing processes
```

Comment:
Does this include ghosts? It's ambiguous if ghosts are 'shared'.

Reply:
Good point.
Since this information is relevant as a map only for owned indices (each index is owned by only one processes and may be shared with another two or so), I think we just need to explain this on documentation and use it consistently.

Reply:
I will rethink some of the naming